logstash.go

```diff
@@ -73,8 +74,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 		}
 		_, err = a.conn.Write(js)
 		if err != nil {
-			log.Println("logstash:", err)
-			continue
+			log.Println("fatal logstash:", err)
+			os.Exit(3)
```

Comment:
There is already a PR that is exactly the same as this. However I was asking the other PR author to use log.Fatalln() instead, but got no reply. Could you change to log.Fatalln() and skip os.Exit(), or is there a reason for error code 3 (instead of 1 in Fatalln)?